localstack-core/localstack/feature_catalog/services/kms_api.py

```diff
@@ -0,0 +1,21 @@
+from localstack.feature_catalog.service_feature import (
+    ApiCoverage,
+    ImplementationStatus,
+    SupportStatus,
+)
+
+
+class CreateKey(ApiCoverage):
```

Comment:
this approach was suggested by @sannya-singal to document additional limitations.
However, we also iterated over it, and came up with a `ApiLimitation` decorator that could take the limitation as input, which may be more practical solution as we don't need to setup subclasses.